daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -64,76 +69,60 @@ The above example uses secrets as plain strings. It is recommended to use a secr
+## Getting Started
 
-### Create consumer groups for each subscriber
+To get started, follow these steps to use the Azure Event Hub component
 
-For every Dapr app that wants to subscribe to events, create an Event Hubs consumer group with the name of the `dapr id`.
-For example, a Dapr app running on Kubernetes with `dapr.io/app-id: "myapp"` will need an Event Hubs consumer group named `myapp`.
+{{% alert title="Warning" color="warning" %}}
+It is important to follow the creation of an Azure Event Hub account correctly as Dapr requires certain aspects to be configured.
+{{% /alert %}}
 
-Note: Dapr passes the name of the Consumer group to the EventHub and so this is not supplied in the metadata.
+### 1. Create an Azure Storage Account
 
-## Entity Management
+Dapr utilizes the Event Processor Host so an Azure Storage Account is required:
 
-When entity management is enabled in configuration, as long as the application has the right role and permissions to manipulate the Event Hub namespace, creation of Event Hubs and consumer groups can be done on the fly. 
+1. Create an [Azure Storage Account](https://docs.microsoft.com/azure/storage/common/storage-account-create?tabs=azure-portal) resource (required since Dapr uses the `Event Processor Host`)
+2. [Copy the Storage Account Keys](https://docs.microsoft.com/azure/storage/common/storage-account-keys-manage)
 
-The Evet Hub name is the `topic` field in the incoming request to publish or subscribe to, while the consumer group name is the name of the `dapr app` which subscribes to a given Event Hub. For example, a Dapr app running on Kubernetes with name `dapr.io/app-id: "myapp"` requires an Event Hubs consumer group named `myapp`.
+### 2. Create and Configure Azure Event Hubs
 
-Entity management is only possible when using [Azure Authentication]({{< ref "authenticating-azure.md" >}}) mechanisms and not via `connectionString`.
+1. Create an [Azure Event Hub Namespace](https://docs.microsoft.com/azure/event-hubs/event-hubs-create) resource
+2. Create an `Event Hub` in the created namespace from step 1
+3. Create a Consumer Group for each app that wants to subscribe
+   1. The name of this consumer group equals the app id passed by the Dapr Run CLI flag `--app-id` or Kubernetes flag `dapr.io/app-id` (e.g. `dapr run --app-id example` or `dapr.io/app-id: "example"` requires a consumer group named `example`)
+4. Decide which authentication mechanism to use (see example below)
+   1. [Azure Authentication]({{< ref authenticating-azure.md >}})
+   2. [Connection String]((https://docs.microsoft.com/azure/event-hubs/authorize-access-shared-access-signature)) (copy the Connection String for the Event Hub from the Namespace or create a Shared Access Policy on the event hub itself (this will then include `EntityPath={EventHub}`).
```

Comment:
```suggestion
   1. [Connection String]((https://docs.microsoft.com/azure/event-hubs/authorize-access-shared-access-signature)); either:
      - Copy the Connection String for the Event Hub from the Namespace, or 
      - Create a Shared Access Policy on the Event Hub itself (this will then include `EntityPath={EventHub}`).
```